rpc/nodeagent/network/logicalnetwork/moc_nodeagent_logicalnetwork.proto

```diff
@@ -27,8 +28,9 @@ message LogicalNetwork {
 	string id = 2;
 	repeated LogicalNetworkIpam ipams = 3;
 	Status status = 4;
-	Entity entity = 5;
-	Tags tags = 6;
+	NetworkType type = 5;
```

Comment:
We should never change the number ordering. It should be added at the end.

Reply:
okay